plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeConfig.java

```diff
@@ -77,6 +77,8 @@
     private boolean uniqueTableLocation = true;
     private boolean legacyCreateTableWithExistingLocationEnabled;
     private boolean registerTableProcedureEnabled;
+    private int readerVersion = 1;
+    private int writerVersion = 2;
```

Comment:
It would be nice to deny setting unsupported versions. You can refer to `MySqlJdbcConfig`.